examples/redis/v1beta3/README.md

```diff
@@ -0,0 +1,118 @@
+
+[redis-controller](redis-controller.yaml)
+
+The bulk of this controller config is actually identical to the redis-master pod definition above.  It forms the template or "cookie cutter" that defines what it means to be a member of this set.
+
+Create this controller:
+
+```sh
+kubectl create -f examples/redis/v1beta3/redis-controller.yaml
+```
+
+We'll do the same thing for the sentinel.  Here is the controller config:
+[redis-sentinel-controller](redis-sentinel-controller.yaml)
+
+We create it as follows:
+```sh
+kubectl create -f examples/redis/v1beta3/redis-sentinel-controller.yaml
+```
+
+### Resize our replicated pods
+Initially creating those pods didn't actually do anything, since we only asked for one sentinel and one redis server, and they already existed, nothing changed.  Now we will add more replicas:
+
+```sh
+kubectl resize rc redis --replicas=3
+```
+
+```sh
+kubectl resize rc redis-sentinel --replicas=3
+```
+
+This will create two additional replicas of the redis server and two additional replicas of the redis sentinel.
+
+Unlike our original redis-master pod, these pods exist independently, and they use the ```redis-sentinel-service``` that we defined above to discover and join the cluster.
+
+### Delete our manual pod
+The final step in the cluster turn up is to delete the original redis-master pod that we created manually.  While it was useful for bootstrapping discovery in the cluster, we really don't want the lifespan of our sentinel to be tied to the lifespan of one of our redis servers, and now that we have a successful, replicated redis sentinel service up and running, the binding is unnecessary.
+
+Delete the master as follows:
+```sh
+kubectl delete pods redis-master
+```
+
+Now let's take a close look at what happens after this pod is deleted.  There are three things that happen:
+
+  1. The redis replication controller notices that its desired state is 3 replicas, but there are currently only 2 replicas, and so it creates a new redis server to bring the replica count back up to 3
+  2. The redis-sentinel replication controller likewise notices the missing sentinel, and also creates a new sentinel.
+  3. The redis sentinels themselves, realize that the master has disappeared from the cluster, and begin the election procedure for selecting a new master.  They perform this election and selection, and chose one of the existing redis server replicas to be the new master.
+
+### Conclusion
+At this point we now have a reliable, scalable Redis installation.  By resizing the replication controller for redis servers, we can increase or decrease the number of read-slaves in our cluster.  Likewise, if failures occur, the redis-sentinels will perform master election and select a new master. 
+
+### tl; dr
+For those of you who are impatient, here is the summary of commands we ran in this tutorial
+
+```sh
+# Create a bootstrap master
+kubectl create -f examples/redis/v1beta3/redis-master.yaml
```

Comment:
Please ensure that all these commands are creating replication controllers, not raw pods.